recipes/rusty-polygon-geohasher/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "rusty-polygon-geohasher" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/rusty_polygon_geohasher-{{ version }}.tar.gz
+  sha256: 6b0935fddc33e5fbeab4e0aa3a6da869e68ebaa64a0126450a7b4299a8957288
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
```

Comment:
You're going to need a `build` section under `requirements` that includes the necessary rust tools. You'll probably want something like:

```
requirements:
   build:
    - ${{ compiler("c") }}
    - ${{ compiler("rust") }}
    - ${{ stdlib("c") }}
    - cargo-bundle-licenses
 ...
```

See also the following about bundling the rust dependency licenses:

https://conda-forge.org/docs/maintainer/adding_pkgs/#rust